lib/tasks/jpets.rake

```diff
@@ -21,6 +21,17 @@ namespace :jpets do
     EmailReminder.special_resend_of_signature_email_validation
   end
 
+  desc "Email members with a petitions report if site is configured to do so today"
+  task :enqueue_petitions_report_if_configured_for_today => :environment do
+    if Time.current.wday == (Site.petition_report_day_of_week + 8 % 7)
+      Task.run("jpets:enqueue_petitions_report_if_configured_for_today") do
+        EmailPetitionsReportJob.set(
+          wait_until: Time.current.change(hour: Site.petition_report_hour_of_day)
+        ).perform_later
+      end
+    end
```

Comment:
I would move this time logic to the Site model, e.g:

``` ruby
if Site.petition_report_due?
  Task.run("jpets:enqueue_petitions_report_if_configured_for_today") do
    EmailPetitionsReportJob.set(wait_until: Site.petitions_report_due_at).perform_later
  end
end
```

And the methods on Site would be:

``` ruby
def petition_report_due?
  Date.current == petition_report_due_at.to_date
end

def petition_report_due_at(sunday = Time.current.beginning_of_week(:sunday))
  (sunday + petition_report_day_of_week.days).change(hour: petition_report_hour_of_day)
end
```